CODE-OWNERS

```diff
@@ -0,0 +1,51 @@
+areas:
+  all:
```

Comment:
good point, i like having all since it's very explicit about the group of people that can cover anything in the code base.  to reduce confusion, i'll remove those names from all the other areas.